65.md

```diff
@@ -6,71 +6,53 @@ Relay List Metadata
-
-Because users don't often share the same sets of relays, ad-hoc solutions have arisen to get that content, but these solutions negatively impact scalability and decentralization:
+The old model of using a fixed relay list per user centralizes in large relay operators: 
 
   - Most people are sending their posts to the same most popular relays in order to be more widely seen
   - Many people are pulling from a large number of relays (including many duplicate events) in order to get more data
   - Events are being copied between relays, oftentimes to many different relays
+  
+This NIP allows apps to connect directly with the most up-to-date relay set from each individual user, eliminating the need to use large operators. 
 
-### Purposes
-
-The purpose of this NIP is to help clients find the events of the people they follow, to help tagged events get to the people tagged, and to help nostr scale better.
-
-### Suggestions
+### Final Considerations
 
-It is suggested that people spread their kind `10002` events to many relays, but write their normal feed-related events to a much smaller number of relays (between 2 to 6 relays). It is suggested that clients offer a way for users to spread their kind `10002` events to many more relays than they normally post to.
+1. Clients should spread an author's kind `10002` events to as many relays as viable, all of them if possible. 
 
-Authors may post events outside of the feed that they wish their followers to follow by posting them to relays outside of those listed in their "Relay List Metadata".  For example, an author may want to reply to someone without all of their followers watching.
+2. Kind `10002` should only be used to advertise relays. The user's configuration when using Clients should stay in Kind 3.
 
-It is suggested that relays allow any user to write their own kind `10002` event (optionally with AUTH to verify it is their own) even if they are not otherwise subscribed to the relay because
-
-  - finding where someone posts is rather important
-  - these events do not have `content` that needs management
-  - relays only need to store one replaceable event per pubkey to offer this service
-
-### Why not in kind `0` Metadata
-
-Even though this is user related metadata, it is a separate event from kind `0` in order to keep it small (as it should be widely spread) and to not have `content` that may require moderation by relay operators so that it is more acceptable to relays.
-
-### Example
-
-```json
-{
-  "kind": 10002,
-  "tags": [
-    ["r", "wss://alicerelay.example.com"],
-    ["r", "wss://brando-relay.com"],
-    ["r", "wss://expensive-relay.example2.com", "write"],
-    ["r", "wss://nostr-relay.example.com", "read"],
-  ],
-  "content": "",
-  ...other fields
-```
+3. DMs should only be broadcasted to each author's WRITE relays to keep maximum privacy. 
```

Comment:
This should be READ relays.  Send to other people at their READ relays, where *they* read from.   Confusing I know and sorry we are stuck with that.

Reply:
Fixed